src/utils/utils.js

```diff
@@ -11,18 +11,143 @@ const disabilityMap = {
   AUTYZM: "z autyzmem, w tym z zespołem Aspergera"
 };
 
-const calculateValues = values => {
-  const reason = values.applicant.reason;
-  let secondReason = values.applicant.secondReason;
-  if (reason && secondReason) {
-    const firstReason = disabilityMap[reason];
-    secondReason = disabilityMap[secondReason];
-    values.applicant.multipleDisability = [firstReason, secondReason];
-    values.applicant.reason = "SPRZEZONA";
-  }
-  return values;
-};
+class DocumentData {
+  constructor(values) {
+    this.values = values;
+  }
+
+  isMultipleDisability() {
+    return Boolean(
+      this.values.applicant.reason && this.values.applicant.secondReason
+    );
+  }
+
+  reason() {
+    if (this.isMultipleDisability()) {
+      return "SPRZEZONA";
+    }
+    return this.values.applicant.reason;
+  }
+
+  multipleDisability() {
+    if (!this.isMultipleDisability) {
+      return [];
+    }
+    const firstReasonDescription = disabilityMap[this.values.applicant.reason];
+    const secondReasonDescription =
+      disabilityMap[this.values.applicant.secondReason];
+    return [firstReasonDescription, secondReasonDescription];
+  }
+
+  applicantsList() {
+    const applicants = [
+      {
+        firstName: this.values.applicant.firstName1,
+        lastName: this.values.applicant.lastName1,
+        address: this.values.applicant.address1,
+        postalCode: this.values.applicant.postalCode1
+      }
+    ];
+    if (this.values.applicant.firstName2) {
+      applicants.push({
+        firstName: this.values.applicant.firstName2,
+        lastName: this.values.applicant.lastName2,
+        address: this.values.applicant.address2,
+        postalCode: this.values.applicant.postalCode2
+      });
+    }
+    return applicants;
+  }
+
+  parentsDescription() {
+    const applicant = this.values.applicant;
+    if (!applicant.firstName2) {
+      return `${applicant.firstName1} ${applicant.lastName1}, ${
+        applicant.address1
+      }, ${applicant.postalCode1}`;
+    }
+    if (applicant.addresFirstParentCheckbox) {
+      return `${applicant.firstName1} ${applicant.lastName1} i ${
+        applicant.firstName2
+      } ${applicant.lastName2}, ${applicant.address1}, ${
+        applicant.postalCode1
+      }`;
+    }
+    return `${applicant.firstName1} ${applicant.lastName1}, ${
+      applicant.address1
+    }, ${applicant.postalCode1} i ${applicant.firstName2} ${
+      applicant.lastName2
+    }, ${applicant.address2}, ${applicant.postalCode2}`;
+  }
+
+  onRequest() {
+    const applicant = this.values.applicant;
+    if (!applicant.firstName2) {
+      return `${applicant.firstName1} ${applicant.lastName1}`;
+    }
+    return `${applicant.firstName1} ${applicant.lastName1} i ${
+      applicant.firstName2
+    } ${applicant.lastName2}`;
+  }
+
+  teamMembers() {
+    return this.values.supportCenter.members;
+  }
+
+  templateData() {
+    return {
+      child: {
+        name: this.values.child.name,
```

Comment:
By any chances to we want to return child object here?, maybe `child: {...this.values.child}` would suffice

Reply:
My goal was to present very clearly and in readable way which keys we are using and which can be removed. Maybe putting some comment in this place to remove that redundant keys after we'll have all the logic here will be enough?